testdata/.tranor/envs.json

```diff
@@ -0,0 +1,14 @@
+[
+	{
+		"name": "dev",
+		"dnsSuffix": "dev.mycompany.com"
+	},
+	{
+		"name": "stg",
+		"dnsSuffix": "stg.mycompany.com"
+	},
+	{
+		"name": "prd",
+		"dnsSuffix": "mycompany.com"
+	}
+]
```

Comment:
I don't think full names are the best solutions - but I agree `stg` and `prd` look weird. I suggest we keep the current naming:
```
- dev
- qa
- stage
- prod
```